src/models/Bookmark.ts

```diff
@@ -64,9 +107,72 @@ export const BookmarkModel = {
       return {
         _id: _id,
       }
-    } catch (e) {
+    } catch (e: any) {
       console.error('Error in Bookmark.hideOne', e)
       return e
     }
   },
+  async addOne(
```

Comment:
this method makes me wonder if we should standardize on a cleaner function signature, for example group the object properties together and pass the context in separately. ie something like:

```
addOne({ collectionId, url, userId, label, cmsId }, { db, ...options })
```

Reply:
updated -- take a look and let me know what you think! ⭐

Reply:
yes!! looks good :D I think it would be nice to roll this out to the other methods as well for the sake of consistency, but not a blocker for this PR

Reply:
yeah happy to do that! wanted to make sure i was on track before doing all of them 😂